### PR TITLE
Addition of Disqus

### PR DESCRIPTION
**Pull request by danward79**
@humboldtux, Added ability to use Disqus comment system. Minor update to readme, to
reflect this.

### CODE REVIEW

File: layouts/about/single.html [hunk 1]
```diff
@@ -21,6 +21,9 @@
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
               {{ .Content }}
+              {{ if .Params.Comments }}
```
Comment (humboldtux) [inline layouts/about/single.html]: Where can i find this parameter `Comments`? in the config file?

Comment (danward79) [inline layouts/about/single.html]: For the about page, it is a front matter param which can be included.

If "Comments" is not present comments do not show. If "Comments" is present then comments show if it is set to true.

`Comments = false` or `Comments = true`

Comment (humboldtux) [inline layouts/about/single.html]: Does this means that comments are enabled by default for all post pages? How to disable comments for all post pages? Disable/Enable them per post page?

Comment (danward79) [inline layouts/about/single.html]: No, in order for comments to work at all (posts or about), the following needs to be included in the config file for the site:

`disqusShortname = "YourDisqusShortname"`

If this is present, comments will show in the post pages, but not the about page.

To enable comments in the about page the front matter needs to include:

`Comments = false` or `Comments = true`

The behaviour could be modified to enable / disable on a per post basis also, that would require an addition to the front matter for posts, which could be facilitated by updating:

- /archetypes/post.md to include:
`Comments = false` or `Comments = true` depending on if the behaviour should be on or off by default.
- /layouts/post/single.html to include:
`{{ if .Params.Comments }}` etc.
 
Q - Would you prefer on a per post basis, so the same behaviour as about?

Comment (danward79) [inline layouts/about/single.html]: Hi @humboldtux, Have you had a chance to consider above?

**Comment by danward79** [comment]
Hi @humboldtux,

That sounds like sensible functionality to me. I'll update the PR to have this functionality.

(ps. I think I missed some of the conversation above. The time stamps are out of order!)

**Comment by danward79** [comment]
@humboldtux, updated as per above.

If no disqus short name provided, then no comments at all.
If disqus short name provided, then comments are added to posts and about. They can be disabled by changing the comments tag in the front matter to false.

The comments tag defaults to true in a new post.

**Comment by humboldtux** [comment]
Thanks

**Pull request merged**